ratis-netty/src/main/java/org/apache/ratis/netty/server/NettyServerStreamRpc.java

```diff
@@ -212,7 +211,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws IOExceptio
 
         JavaUtils.allOf(remoteWrites).thenCombine(localWrite, (v, bytesWritten) -> {
               buf.release();
-              sendReply(request, ctx);
+              sendReply(request, bytesWritten, ctx);
```

Comment:
It might be useful to check whether `bytesWritten` of local write is equal to `bytesWritten` to peers as a correct check.

Reply:
That is a good idea.  When bytesWritten is different, it should retry to send the remaining data.  Let's work on it separately since it needs some works.

Reply:
SG thanks!